Pinta/Actions/File/SaveDocumentImplementationAction.cs

```diff
@@ -189,6 +189,29 @@ private bool SaveFile (Document document, Gio.File? file, FormatDescriptor? form
 			return false;
 		}
 
+		// If the format doesn't support layers, ask to flatten the image
+		if (!format.SupportsLayers && document.Layers.Count() > 1)
```

Comment:
Having this prompt always show up if there are multiple layers might be too annoying / confusing for common cases where users just want to save to a .png / jpeg and don't care about preserving layers.
I think limiting it to only appearing if the previous file type did support layers (e.g. `.ora`) would probably be a safer starting point

Reply:
i agree prompting might be a bit over the top in that case.
Should we still execute the flatten action though?

Reply:
In that case I'd probably skip the flatten action and just do it implicitly, so the behaviour is unchanged for the common case